packages/govuk-frontend/src/govuk/components/task-list/accessibility.test.mjs

```diff
@@ -0,0 +1,22 @@
+import { axe, goToComponent } from 'govuk-frontend-helpers/puppeteer'
+import { getExamples } from 'govuk-frontend-lib/files'
+
+describe('/components/task-list', () => {
+  describe('component examples', () => {
+    let exampleNames
+
+    beforeAll(async () => {
+      exampleNames = Object.keys(await getExamples('task-list'))
+    })
+
+    it('passes accessibility tests', async () => {
+      for (const name of exampleNames) {
+        const exampleName = name.replace(/ /g, '-')
+
+        // Navigation to example, create report
+        await goToComponent(page, 'task-list', { exampleName })
```

Comment:
Thanks for adding these new tests 😊

Reply:
Looks like they've started failing though (since the merge from main) for some reason. Will investigate!

Reply:
For the Cheerio type checks in `template.test.js` [try these ideas](https://github.com/alphagov/govuk-frontend/pull/2261#discussion_r1203793390)

You can run them locally with:

```console
npm run build:types
```

Or for accessibility tests in particular we find [opening the browser](https://github.com/alphagov/govuk-frontend/blob/main/docs/releasing/testing-and-linting.md#component-tests) is helpful:

```console
HEADLESS=false npx jest task-list/accessibility.test.mjs --watch
```

But for tests in a loop like these you'll need to pause Jest or the browser will close:

```js
await expect(axe(page)).resolves.toHaveNoViolations()
await jestPuppeteer.debug() // Pause the test run for debugging
```